include/fmt/format.h

```diff
@@ -940,8 +940,8 @@ template <typename T = void> struct FMT_EXTERN_TEMPLATE_API basic_data {
   static const char reset_color[5];
   static const wchar_t wreset_color[5];
   static const char signs[];
-  static constexpr const char left_padding_shifts[] = {31, 31, 0, 1, 0};
-  static constexpr const char right_padding_shifts[] = {0, 31, 0, 1, 0};
+  static constexpr const char left_padding_shifts[5] = {31, 31, 0, 1, 0};
+  static constexpr const char right_padding_shifts[5] = {0, 31, 0, 1, 0};
```

Comment:
Some GCC versions fail to determine the size of this array - https://godbolt.org/z/f4s98c

Reply:
This is a gcc bug https://gcc.gnu.org/bugzilla/show_bug.cgi?id=66921.